### PR TITLE
Refresh bundles automatically

### PR DESCRIPTION
**Pull request by ChengshuLi**
Fixes codalab/codalab-worksheets#104

@percyliang

### CODE REVIEW

File: codalab/rest/legacy.py [hunk 1]
```diff
@@ -208,10 +211,10 @@ def get_and_update_summary(download_manager, item, name):
                 with info_lock:
                     info[name] = result
 
-            read_threads = []            
+            read_threads = []
             for item in info['contents']:
                 name = item['name']
-                if name in ['stdout', 'stderr'] and (item['type'] == 'file' or item['type'] == 'link'):
+                if name in ['stdout', 'stderr'] or (item['type'] == 'file' or item['type'] == 'link'):
```
Comment (percyliang) [inline codalab/rest/legacy.py:217]: why this change?

**Comment by percyliang** [comment]
@ChengshuLi could you explain what this PR is doing?

**Comment by ChengshuLi** [comment]
@percyliang 
I want to periodically update run bundle, including all the embedded file, so I need to fetch the file contents for that.

Here is an example.

% schema foo
% add myoutput  /output
% display table foo
[run run-while : while true; do echo 1 >> output; sleep 0.2; done]

In this case, I need to update the /output file in the bundle so I need to fetch its content

**Comment by percyliang** [comment]
Did you check that there's not a performance hit?

File: codalab/rest/legacy.py [hunk 2]
```diff
@@ -184,7 +187,7 @@ def get_summary(download_manager, info, name):
                     '\n')
                 contents = download_manager.summarize_file(
                     uuid, name,
-                    num_head_lines=50, num_tail_lines=50, max_line_length=128,
+                    num_head_lines=1000, num_tail_lines=0, max_line_length=128,
```
Comment (klopyrev) [inline codalab/rest/legacy.py:190]: Why are you doing this?

Comment (klopyrev) [inline codalab/rest/legacy.py:190]: It's going to be a lot of data for a running bundle.

**Comment by ChengshuLi** [comment]
Close comment: I completely re-did this PR.
